lab_2/main.py

```diff
@@ -24,7 +32,18 @@ def create_zero_matrix(rows: int, columns: int) -> list:
     e.g. rows = 2, columns = 2
     --> [[0, 0], [0, 0]]
     """
-    pass
+    if isinstance(rows, int) and isinstance(columns, int):
+        if rows == 1 and columns == 1:
+            table = [[1]]
+        else:
+            table = []
+            column = []
+            for i in range(columns):
+                column.append(0)
+            for n in range(rows):
+                table.append(column)
```

Comment:
can you explain this implementation, please?

Reply:
I changed this realisation.